File: Iniciante/URI 1013.cs
using System; 
 
class URI {
 
    static void Main(string[] args) { 
		string[] input = Console.ReadLine().Split(' ');
		int valor1 = int.Parse(input[0]);
		int valor2 = int.Parse(input[1]);
		int valor3 = int.Parse(input[2]);
		if (valor1 > valor2 && valor1 > valor3) {
			Console.WriteLine("{0} eh o maior", valor1);
		} else if (valor2 > valor3) {
			Console.WriteLine("{0} eh o maior", valor2);
		} else {
			Console.WriteLine("{0} eh o maior", valor3);
		}
    }
    
}

File: Iniciante/URI 1013.py
valor1 = None
valor2 = None
valor3 = None
lista = None

def read_line():
  try:
    # read for Python 2.x
    return raw_input()
  except NameError:
    # read for Python 3.x
    return input()

lista = read_line().split(" ")
valor1 = int(lista[0])
valor2 = int(lista[1])
valor3 = int(lista[2])

if valor1 > valor2 and valor1 > valor3:
	print(str(valor1) + " eh o maior")
elif valor2 > valor3:
	print(str(valor2) + " eh o maior")
else:
	print(str(valor3) + " eh o maior")